starters/tailwind/src/Disclosure.tsx

```diff
@@ -0,0 +1,106 @@
+import React, { useContext } from "react";
+import {
+  UNSTABLE_Disclosure as AriaDisclosure,
+  UNSTABLE_DisclosureGroup as AriaDisclosureGroup,
+  DisclosureProps as AriaDisclosureProps,
+  DisclosureGroupProps as AriaDisclosureGroupProps,
+  UNSTABLE_DisclosurePanel as AriaDisclosurePanel,
+  DisclosurePanelProps as AriaDisclosurePanelProps,
+  composeRenderProps,
+  Heading,
+  Button,
+  DisclosureStateContext,
+} from "react-aria-components";
+import { tv } from "tailwind-variants";
+import { ChevronRight } from "lucide-react";
+import { composeTailwindRenderProps, focusRing } from "./utils";
+import { DisclosureGroupStateContext } from "react-aria-components";
+
+const disclosure = tv({
+  base: "w-64 border rounded-lg bg-gray-100 text-gray-600 border-gray-200 hover:border-gray-300 dark:bg-zinc-700 dark:text-zinc-300 dark:border-zinc-600 dark:hover:border-zinc-500 outline-none",
+});
+
+const disclosureInGroup = tv({
+  extend: disclosure,
+  base: "border-0 border-b last:border-b-0 rounded-b-none last:rounded-b-lg",
```

Comment:
Would be nice to get no bottom border radius on all but the last one, but doesn't seem to be working

Reply:
Looks like its working for me now.